.gitignore

```diff
@@ -269,6 +269,7 @@ devpkg/*
 *.exp
 warzone*.*build
 *.lnk
+CMakeSettings.json
```

Comment:
```suggestion
/CMakeSettings.json
/launch.vs.json
```

Reply:
that works for me too. concerning that launch.vs.json, i think that is not stored in the repo because i have set up one with 3-4 different run configurations and git hasnt complained about it yet. doesnt hurt to have it in there in any case.